ci/requirements/min-all-deps.yml

```
@@ -10,46 +10,45 @@ dependencies:
   - python=3.8
   - boto3=1.13
   - bottleneck=1.3
-  # cartopy 0.18 conflicts with pynio
-  - cartopy=0.17
+  - cartopy=0.19
   - cdms2=3.1
   - cfgrib=0.9
-  - cftime=1.2
+  - cftime=1.4
   - coveralls
-  - dask-core=2.30
-  - distributed=2.30
-  - h5netcdf=0.8
+  - dask-core=2021.04
+  - distributed=2021.04
+  - h5netcdf=0.11
   - h5py=2.10
   # hdf5 1.12 conflicts with h5py=2.10
   - hdf5=1.10
```

```suggestion
  - h5py=3.1
  # hdf5 1.12 conflicts with h5py=3.1
  - hdf5=1.10
```
Sorry I accidentally changed this on your PR instead of mine :disappointed: , this reverts it.